src/main/java/ru/ayaz/MessageParser.java

```diff
@@ -1,6 +1,6 @@
 package ru.ayaz;
 
-public class MessageProcessor {
+public class MessageParser {
```

Comment:
&#x1F34E; **Issue**
If this class has no state (meaning its not an object, its just a set of functions) then you should make its methods static and treat it just a a bunch of utility functions.